test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/V2/ExceptionConvertors/SystemExceptionConvertorTest.cs

```diff
@@ -95,6 +95,7 @@ public class SystemExceptionConvertorTest
             new AbandonedMutexException("AbandonedMutexException"),
             new SemaphoreFullException("SemaphoreFullException"),
             new SynchronizationLockException("SynchronizationLockException"),
+            new TaskCanceledException("TaskCanceledException"),
```

Comment:
Well, on the plus side, these tests actually have some parameter validation logic, but there are several anti-patterns. 
- Instead of loops, we can use xUnit Theory (parameterized test methods)
- Instead of ifs, we should split the tests into functions so that conditions aren't needed.